sys-cluster/glusterfs/glusterfs-4.1.5.ebuild

```diff
@@ -0,0 +1,247 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python2_7 )
+
+inherit autotools elisp-common python-single-r1 systemd user versionator
+
+if [[ ${PV#9999} != ${PV} ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/gluster/glusterfs.git"
+else
+	SRC_URI="https://download.gluster.org/pub/gluster/${PN}/$(get_version_component_range '1-2')/${PV}/${P}.tar.gz"
+	KEYWORDS="~amd64 ~arm ~arm64 ~ppc ~ppc64 ~x86"
+fi
+
+DESCRIPTION="GlusterFS is a powerful network/cluster filesystem"
+HOMEPAGE="https://www.gluster.org/"
+
+LICENSE="|| ( GPL-2 LGPL-3+ )"
+SLOT="0"
+IUSE="
```

Comment:
I would suggest against this, personally. also, while you have it like this, you
may as well sort it alphabetically.

Reply:
Thanks for your answer!

Are you talking about LICENSE? This string was copied from 9999 ebuild. What exactly would you like to see here?

Reply:
Talking about USE=flag\nflag\nflag\n

Reply:
But it's more readable and works well. Is there any technical reason not to use multiline string?
Looks like list is sorted already.
Would you like to see something like this?
```
IUSE="
	bd-xlator
	crypt-xlator
	debug
	emacs
	glupy
	infiniband
	ipv6
	libressl
	qemu-block
	rsyslog
	static-libs
	systemtap
	test
	vim-syntax
	+fuse
	+georeplication
	+libtirpc
	+syslog
	+tiering
	+xml
"
```

Reply:
The majority can be wrong.
My points for multilines:
 - they are readable
 - sorting them as simple as typing `:!sort` in vim line selection mode

But, of course, I can do nothing against the rules. But on my discretion there are no infringements.

Reply:
@eugene-bright True. I tend to stick to existing styles but those are some
decent points. I'd stick with what's there and perhaps file a bug about it,
get a concensus. Also, gs<num>j is really good if you have the sort-motion
plugin installed.

Reply:
I think sort-motion uses vim's internal sort algorithm.
`:sort` and `:!sort` produce distinct results. The last one is more suitable here.
Does Gentoo have something like Python's PEP8?

Reply:
Rule of thumb: you touch an ebuild that is not maintained by you, you stick to its coding style.

Reply:
That's may be one reason among the many why no one has been touched it for months.
I propose a new [PEP8 like GLEP development][1].
For now I revert style changes to facilitate changes acceptance. Users are waiting for them.

[1]: https://bugs.gentoo.org/669848

Reply:
> That's may be one reason among the many why no one has been touched it for months.

I'm going to be bold and say no.

Reply:
One day [cargo cult will destroy our planet][1].

[1]: https://collapseofindustrialcivilization.com/tag/cargo-cults/